server/etcdmain/help.go

```diff
@@ -239,6 +239,10 @@ Experimental feature:
     Enable to check data corruption before serving any client/peer traffic.
   --experimental-corrupt-check-time '0s'
     Duration of time between cluster corruption check passes.
+  --experimental-compact-hash-check-enabled 'false'
```

Comment:
Suggest we keep the changes specifically focused on backporting the workflows only, I'm unsure why this has been included.